### PR TITLE
refactor(cleanup): Code Revamp

### PR DESCRIPTION
**Pull request by tomerlichtash**
- Style revamp WIP
- Button mixin related fixes
- Mixins cleanup WIP
- Cleanup continue
- Mixins cleanup
- Blog design WIP
- Reorganize file structure
- Blog design with PageBase
- Theme Popover WIP
- Updare Stylable v5.1.0
- Loading indicator theming
- Update mixins path
- Fix theme selection
- Fix custom document and theme
- Remove unused code
- Remove features QueryContext and CopyUrlButton

### CODE REVIEW

**Comment by tomerlichtash** [comment]
Regarding Empty `getStaticProps` in static pages - See references:
* https://github.com/vercel/next.js/discussions/18235
* https://eric-schaefer.com/til/2022/02/05/next.js-unexpected-missing-ctx.req-in-_document/

File: src/components/content/content-blocks/code-block/code-block.tsx [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-	className
+	className,
 }: ContentComponentProps): JSX.Element => {
 	const { node } = componentData;
 	return (
-		<code className={st(classes.root, className)} key={node.key}>
-			{node.text}
-		</code>
+		<div className={st(classes.root, className)}>
+			<pre className={classes.pre}>
```
Comment (tomerlichtash) [inline src/components/content/content-blocks/code-block/code-block.tsx:16]: Do we need `pre`?

File: src/components/content/generic-page/generic-page.tsx [hunk 2]
```diff
@@ -29,10 +30,10 @@ export default function GenericPage(props: IContentComponentData) {
 			<Head>
 				{siteTitle} - {siteSubtitle} - {metaData?.title}
 			</Head>
-			<article className={classes.root}>
+			<article className={st(classes.root, { textDirection })}>
```
Comment (tomerlichtash) [inline src/components/content/generic-page/generic-page.tsx:33]: Do we need `textDirection`?

File: src/components/radix-primitives/popover/popover.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import * as RadixPopover from "@radix-ui/react-popover";
-import { createCompRef } from "../../create-ref";
+import { createCompRef } from "../create-ref";
 
 export const PopoverRoot = createCompRef(RadixPopover.Root);
 export const PopoverTrigger = createCompRef(RadixPopover.Trigger);
 export const PopoverContent = createCompRef(RadixPopover.Content);
 export const PopoverClose = createCompRef(RadixPopover.Close);
 export const PopoverArrow = createCompRef(RadixPopover.Arrow);
+export const PopoverPortal = RadixPopover.Portal;
```
Comment (tomerlichtash) [inline src/components/radix-primitives/popover/popover.ts:9]: Do we need PopoverPortal?

Comment (tomerlichtash) [reply src/components/radix-primitives/popover/popover.ts:9]: We don't. Remove.

File: src/consts.ts [hunk 4]
```diff
@@ -9,3 +9,6 @@ export enum CONTENT_TYPES {
 	GLOSSARY = "glossary",
 	ANNOTATION = "annotations",
 }
+
+export const PUBLIC_PROJECT_GITHUB_ADDRESS =
```
Comment (tomerlichtash) [inline src/consts.ts:13]: Rename: PUBLIC_PROJECT_ADDRESS

File: src/contexts/theme-context.tsx [hunk 5]
```diff
@@ -1,77 +1,50 @@
-import React, { Context, createContext, useRef } from "react";
+import React, { Context, createContext } from "react";
```
Comment (tomerlichtash) [inline src/contexts/theme-context.tsx:1]: Use themes.ts

**Review by imdfl — APPROVED** [review]
292 tests performed by 8 workers. As if Abe Lincoln never emancipated the slaves

**Pull request merged**